.github/dependabot.yml

```diff
@@ -15,6 +15,10 @@ updates:
     labels:
       - "dependencies"
       - "bot"
+    ignore:
+      #TODO wait until we are building with Java 21 and Asciidoctor is updated to support jruby 10
+      - dependency-name: "org.jruby:jruby"
+      - versions: [ ">10.0.0.0" ]
```

Comment:
With https://docs.github.com/en/code-security/dependabot/working-with-dependabot/dependabot-options-reference#versions-ignore shouldn't it be something like 
```yml
ignore:
  - dependency-name: "..."
    versions: [10,)
```
? (Note: in my comment the `versions` is not introduced with dash as new element.)

Isn't the current version going to ignore BOTH - dependencies if named - as specified, OR in versions as specified? (+I'm unfamiliar with '>10.0.0.0' syntax for maven versions.)

Reply:
Wow good catch.  I must have been looking at the syntax documentation for the wrong type of package manager.  Fixed.